hydromt_wflow/version_upgrade.py

```diff
@@ -130,6 +130,12 @@ def _convert_to_wflow_v1(
     config_out: dict
         The converted config.
     """
+    if config["model"].get("reinit") == False:
+        logger.warning(
+            "The 'reinit' option set to False is no longer supported in Wflow v1. "
+            "It will be converted to 'cold_start__flag = True'."
```

Comment:
I would change this message into something like: "Converting states is not supported by this conversion code, therefore the `reinit` option (new name `cold_start__flag`) is adjusted to `True`. If model states are required, please produce new states using a Wflow simulation or use the `setup_cold_states` function."